tests/phpunit/tests/option/wpPrimeNetworkOptionCaches.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Test wp_prime_network_option_caches().
+ *
+ * @group option
+ *
+ * @covers ::wp_prime_network_option_caches
+ */
+class Tests_Option_WpPrimeNetworkOptionCaches extends WP_UnitTestCase {
+
+	/**
+	 * Tests that wp_prime_network_option_caches() primes multiple options.
+	 *
+	 * @ticket 61053
+	 */
+	public function test_wp_prime_network_option_caches() {
+		// Create some options to prime.
+		$options_to_prime = array(
+			'option1',
+			'option2',
+			'option3',
+		);
+
+		/*
+		 * Set values for the options,
+		 * clear the cache for the options,
+		 * check options are not in cache initially.
+		 */
+		foreach ( $options_to_prime as $option ) {
+			update_network_option( null, $option, "value_$option" );
+		}
```

Comment:
`update_network_option()` primes the cache so the getter calls below won't trigger DB queries if the priming function fails.

You'll need to do something similar to the test for `wp_prime_option_caches()`

https://github.com/WordPress/wordpress-develop/blob/17a01ed185f38f87362eb7588908dc95ec6cbe23/tests/phpunit/tests/option/wpPrimeOptionCaches.php#L24-L33

:1234: This applies to the other tests too.